newIDE/app/src/ProjectManager/index.js

```diff
@@ -308,7 +304,7 @@ export default class ProjectManager extends React.Component<Props, State> {
   _addEventsFunctionsExtension = (index: number) => {
     const { project } = this.props;
 
-    const newName = newNameGenerator('NewExtension', name =>
+    const newName = newNameGenerator('UntitledExtension', name =>
```

Comment:
`Untitled extension` to be consistent with the other names?

Reply:
Extension names must be valid "identifiers" (in the sense: usable in expressions), so like objects can't contain space.
This could be "fixed" by showing the extension full name (the user friendly name), but that would also hide this "identifier" that is used in expressions. For now I'll keep it this way!